### ISSUE TITLE
Docs: space import command, space-id description missleading

### ISSUE THREAD

**Issue by rubiesonthesky** [contentful/contentful-cli#18]
https://github.com/contentful/contentful-cli/tree/master/docs/space/import

In usage section the doc says

```
Options:

  --space-id              ID of Space with source data       [string] [required]
                          (if not specified it will take the context active space)
```

Shouldn't it be the ID of Space where you want to import the data?

**Comment by floelhoeffel** [comment]
Thanks for flagging @rubiesonthesky !

@axe312ger are those generate? Otherwise I would fix.

**Comment by axe312ger** [comment]
Nope they are not, the options section is basically a copy & paste from the help command. The actual help for the import command contains the proper description, already. (https://github.com/contentful/contentful-cli/blob/master/lib/cmds/space_cmds/import.js#L15)

So yes, feel free to update the doc :)

**Comment by loudmouth** [comment]
This was fixed in #20
